tests/python/test_with_sklearn.py

```diff
@@ -28,23 +27,23 @@ def __exit__(self, exc_type, exc_value, traceback):
         shutil.rmtree(self.name)
 
 
-def test_binary_classification():
+@pytest.mark.parametrize('cls', [xgb.XGBClassifier, xgb.XGBRFClassifier])
```

Comment:
Do we need `ids` in here?

Reply:
Not strictly necessary, `ids` produce nicer outputs in the test listing